service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java

```diff
@@ -168,4 +179,57 @@ public void inviteToTeam(JWTData userData, InviteMembersRequest inviteMembersReq
       emailer.sendInviteEmail(email, "Team Member", inviter, inviterTeam);
     });
   }
+
+  @Override
+  public GetAllTeamsResponse getAllTeams() {
+    List<Team> teamPojos = db.selectFrom(TEAM).fetchInto(Team.class);
+    List<TeamSummary> teams = teamPojos.stream().map(
+        team -> new TeamSummary(team.getId(), team.getName(), getTeamMembers(team.getId()).size()))
+        .collect(Collectors.toList());
+    return new GetAllTeamsResponse(teams, teams.size());
+  }
+
+  @Override
+  public TeamResponse getSingleTeam(int teamId) {
+    Team teamPojo = db.selectFrom(TEAM).where(TEAM.ID.eq(teamId)).fetchOneInto(Team.class);
```

Comment:
Add a null check here, if we're given an invalid teamId we should have a custom exception to check for that